gsoc/proposal-EfficientDataStorage.md

```diff
@@ -0,0 +1,112 @@
+## Google Summer of Code - Application Template
+
+### Student
+
+**Name:** Chaitanya Prakash Bapat, Chaitanya
+
+**Affiliation:** Sardar Patel Institute of Technology (University of Mumbai), Computer Engineering, Bachelor of Engineering (BE)
+
+**Location:** Mumbai, India and India Time Zone UTC+05:30
+
+**GitHub handle:** [@ChaiBapchya](https://github.com/ChaiBapchya)
+
+**Slack handle:** @chaitanyabapat
+
+**Email:** chai.bapat@gmail.com
+
+**Other Contact:**  
+
+**Twitter** : [@ChaiBapchya](https://twitter.com/ChaiBapchya)
+
+**LinkedIn** : [/in/chaitanyaprakashbapat](https://www.linkedin.com/in/chaitanyaprakashbapat)
+
+### Project Details
+
+**Project Title:**  
+
+Efficient Data Storage using Geo-temporal Clustering and Diff-based Storage
+
+**Project Abstract/Summary:**  
+
+Clustering the web pages (data) based on Geographical location and time in order to facilitate Machine Learning analysis.
+
+**Describe the need your project fills:**  
+
+_Broader Goal_ :- Better data management and a step towards creating Environment Data Warehouse
+
+The project will fulfill the following needs - 
+
```

Comment:
To extend that ---
a priority for us is to 
1: build a tool that can replicate the workflow analysts are currently using google sheets and the Versionista scripts for
2: transfer over the annotations analysts have made on existing diffs
3: support/streamline their new work through implement basic clustering and prioritization of changes 
4: investigate how to best annotate/understand/prioritize the changes in the clusters, using NLP and/or ML